benchmarks/different_model_options.py

```diff
@@ -139,12 +139,12 @@ class TimeBuildModelSEI:
             "none",
             "constant",
             "reaction limited",
-            "reaction limited (asymmetric)",
+            # "reaction limited (asymmetric)",
             "solvent-diffusion limited",
             "electron-migration limited",
             "interstitial-diffusion limited",
             "ec reaction limited",
-            "ec reaction limited (asymmetric)",
+            # "ec reaction limited (asymmetric)",
```

Comment:
Commented out for now as both of them fail with both `SPM` and `DFN` with the [error](https://pipelines.actions.githubusercontent.com/serviceHosts/010a1a2d-77c2-43e2-ac6d-6216bf3ea615/_apis/pipelines/1/runs/13853/signedlogcontent/2?urlExpires=2023-04-07T15%3A15%3A20.7397285Z&urlSigningMethod=HMACV1&urlSignature=fSZkRZuPucTQWqpfPGDdiw%2FabByU62TVI0AeMncIHLQ%3D) -

```
For parameters: <class 'pybamm.models.full_battery_models.lithium_ion.spm.SPM'>, 'reaction limited (asymmetric)'
Traceback (most recent call last):
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 601, in process_symbol
    return self._processed_symbols[symbol]
KeyError: Negate(0x3387ad1e3bdca904, -, children=['Inner SEI partial molar volume [m3.mol-1] * x-average(Inner SEI reaction proportion * exp((SEI growth activation energy [J.mol-1] / Ideal gas constant [J.K-1.mol-1]) * ((1.0 / Reference temperature [K]) - (1.0 / Ambient temperature [K]))) * -SEI reaction exchange current density [A.m-2] * exp(-SEI growth transfer coefficient * Faraday constant [C.mol-1] / (Ideal gas constant [J.K-1.mol-1] * Ambient temperature [K]) * (broadcast((2.0 * Ideal gas constant [J.K-1.mol-1] * Ambient temperature [K] / Faraday constant [C.mol-1]) * arcsinh(Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m]) / (2.0 * Negative electrode exchange-current density [A.m-2])) + Negative electrode OCP [V] + 1e-06 * (1.0 / (maximum(minimum(boundary value(X-averaged negative particle concentration [mol.m-3]) / Maximum concentration in negative electrode [mol.m-3], 0.9999999999), 1e-10)) + 1.0 / (-1.0 + maximum(minimum(boundary value(X-averaged negative particle concentration [mol.m-3]) / Maximum concentration in negative electrode [mol.m-3], 0.9999999999), 1e-10))) + (Ambient temperature [K] - Reference temperature [K]) * Negative electrode OCP entropic change [V.K-1] - (-Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m]) * (X-averaged inner SEI thickness [m] + X-averaged outer SEI thickness [m]) * SEI resistivity [Ohm.m]) - SEI open-circuit potential [V]) - (broadcast(Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m])) * (broadcast(X-averaged inner SEI thickness [m]) + broadcast(X-averaged outer SEI thickness [m])) * SEI resistivity [Ohm.m])))) / (Faraday constant [C.mol-1] * Ratio of lithium moles to SEI moles)'], domains={'primary': ['current collector']})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 601, in process_symbol
    return self._processed_symbols[symbol]
KeyError: Division(-0x4d47f7609b7fb8b1, /, children=['Inner SEI partial molar volume [m3.mol-1] * x-average(Inner SEI reaction proportion * exp((SEI growth activation energy [J.mol-1] / Ideal gas constant [J.K-1.mol-1]) * ((1.0 / Reference temperature [K]) - (1.0 / Ambient temperature [K]))) * -SEI reaction exchange current density [A.m-2] * exp(-SEI growth transfer coefficient * Faraday constant [C.mol-1] / (Ideal gas constant [J.K-1.mol-1] * Ambient temperature [K]) * (broadcast((2.0 * Ideal gas constant [J.K-1.mol-1] * Ambient temperature [K] / Faraday constant [C.mol-1]) * arcsinh(Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m]) / (2.0 * Negative electrode exchange-current density [A.m-2])) + Negative electrode OCP [V] + 1e-06 * (1.0 / (maximum(minimum(boundary value(X-averaged negative particle concentration [mol.m-3]) / Maximum concentration in negative electrode [mol.m-3], 0.9999999999), 1e-10)) + 1.0 / (-1.0 + maximum(minimum(boundary value(X-averaged negative particle concentration [mol.m-3]) / Maximum concentration in negative electrode [mol.m-3], 0.9999999999), 1e-10))) + (Ambient temperature [K] - Reference temperature [K]) * Negative electrode OCP entropic change [V.K-1] - (-Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m]) * (X-averaged inner SEI thickness [m] + X-averaged outer SEI thickness [m]) * SEI resistivity [Ohm.m]) - SEI open-circuit potential [V]) - (broadcast(Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m])) * (broadcast(X-averaged inner SEI thickness [m]) + broadcast(X-averaged outer SEI thickness [m])) * SEI resistivity [Ohm.m]))))', 'Faraday constant [C.mol-1] * Ratio of lithium moles to SEI moles'], domains={'primary': ['current collector']})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 601, in process_symbol
    return self._processed_symbols[symbol]
KeyError: Multiplication(-0x6c16b816a50a0fa8, *, children=['Inner SEI partial molar volume [m3.mol-1]', 'x-average(Inner SEI reaction proportion * exp((SEI growth activation energy [J.mol-1] / Ideal gas constant [J.K-1.mol-1]) * ((1.0 / Reference temperature [K]) - (1.0 / Ambient temperature [K]))) * -SEI reaction exchange current density [A.m-2] * exp(-SEI growth transfer coefficient * Faraday constant [C.mol-1] / (Ideal gas constant [J.K-1.mol-1] * Ambient temperature [K]) * (broadcast((2.0 * Ideal gas constant [J.K-1.mol-1] * Ambient temperature [K] / Faraday constant [C.mol-1]) * arcsinh(Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m]) / (2.0 * Negative electrode exchange-current density [A.m-2])) + Negative electrode OCP [V] + 1e-06 * (1.0 / (maximum(minimum(boundary value(X-averaged negative particle concentration [mol.m-3]) / Maximum concentration in negative electrode [mol.m-3], 0.9999999999), 1e-10)) + 1.0 / (-1.0 + maximum(minimum(boundary value(X-averaged negative particle concentration [mol.m-3]) / Maximum concentration in negative electrode [mol.m-3], 0.9999999999), 1e-10))) + (Ambient temperature [K] - Reference temperature [K]) * Negative electrode OCP entropic change [V.K-1] - (-Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m]) * (X-averaged inner SEI thickness [m] + X-averaged outer SEI thickness [m]) * SEI resistivity [Ohm.m]) - SEI open-circuit potential [V]) - (broadcast(Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m])) * (broadcast(X-averaged inner SEI thickness [m]) + broadcast(X-averaged outer SEI thickness [m])) * SEI resistivity [Ohm.m]))))'], domains={'primary': ['current collector']})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 601, in process_symbol
    return self._processed_symbols[symbol]
KeyError: XAverage(0x11c7235ae080326b, x-average, children=['Inner SEI reaction proportion * exp((SEI growth activation energy [J.mol-1] / Ideal gas constant [J.K-1.mol-1]) * ((1.0 / Reference temperature [K]) - (1.0 / Ambient temperature [K]))) * -SEI reaction exchange current density [A.m-2] * exp(-SEI growth transfer coefficient * Faraday constant [C.mol-1] / (Ideal gas constant [J.K-1.mol-1] * Ambient temperature [K]) * (broadcast((2.0 * Ideal gas constant [J.K-1.mol-1] * Ambient temperature [K] / Faraday constant [C.mol-1]) * arcsinh(Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m]) / (2.0 * Negative electrode exchange-current density [A.m-2])) + Negative electrode OCP [V] + 1e-06 * (1.0 / (maximum(minimum(boundary value(X-averaged negative particle concentration [mol.m-3]) / Maximum concentration in negative electrode [mol.m-3], 0.9999999999), 1e-10)) + 1.0 / (-1.0 + maximum(minimum(boundary value(X-averaged negative particle concentration [mol.m-3]) / Maximum concentration in negative electrode [mol.m-3], 0.9999999999), 1e-10))) + (Ambient temperature [K] - Reference temperature [K]) * Negative electrode OCP entropic change [V.K-1] - (-Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m]) * (X-averaged inner SEI thickness [m] + X-averaged outer SEI thickness [m]) * SEI resistivity [Ohm.m]) - SEI open-circuit potential [V]) - (broadcast(Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m])) * (broadcast(X-averaged inner SEI thickness [m]) + broadcast(X-averaged outer SEI thickness [m])) * SEI resistivity [Ohm.m])))'], domains={'primary': ['current collector']})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 601, in process_symbol
    return self._processed_symbols[symbol]
KeyError: Multiplication(-0x172720f67329df34, *, children=['Inner SEI reaction proportion * exp((SEI growth activation energy [J.mol-1] / Ideal gas constant [J.K-1.mol-1]) * ((1.0 / Reference temperature [K]) - (1.0 / Ambient temperature [K])))', '-SEI reaction exchange current density [A.m-2] * exp(-SEI growth transfer coefficient * Faraday constant [C.mol-1] / (Ideal gas constant [J.K-1.mol-1] * Ambient temperature [K]) * (broadcast((2.0 * Ideal gas constant [J.K-1.mol-1] * Ambient temperature [K] / Faraday constant [C.mol-1]) * arcsinh(Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m]) / (2.0 * Negative electrode exchange-current density [A.m-2])) + Negative electrode OCP [V] + 1e-06 * (1.0 / (maximum(minimum(boundary value(X-averaged negative particle concentration [mol.m-3]) / Maximum concentration in negative electrode [mol.m-3], 0.9999999999), 1e-10)) + 1.0 / (-1.0 + maximum(minimum(boundary value(X-averaged negative particle concentration [mol.m-3]) / Maximum concentration in negative electrode [mol.m-3], 0.9999999999), 1e-10))) + (Ambient temperature [K] - Reference temperature [K]) * Negative electrode OCP entropic change [V.K-1] - (-Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m]) * (X-averaged inner SEI thickness [m] + X-averaged outer SEI thickness [m]) * SEI resistivity [Ohm.m]) - SEI open-circuit potential [V]) - (broadcast(Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m])) * (broadcast(X-averaged inner SEI thickness [m]) + broadcast(X-averaged outer SEI thickness [m])) * SEI resistivity [Ohm.m])))'], domains={'primary': ['negative electrode'], 'secondary': ['current collector']})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 601, in process_symbol
    return self._processed_symbols[symbol]
KeyError: Multiplication(0x1ce327ad1a8100d7, *, children=['-SEI reaction exchange current density [A.m-2]', 'exp(-SEI growth transfer coefficient * Faraday constant [C.mol-1] / (Ideal gas constant [J.K-1.mol-1] * Ambient temperature [K]) * (broadcast((2.0 * Ideal gas constant [J.K-1.mol-1] * Ambient temperature [K] / Faraday constant [C.mol-1]) * arcsinh(Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m]) / (2.0 * Negative electrode exchange-current density [A.m-2])) + Negative electrode OCP [V] + 1e-06 * (1.0 / (maximum(minimum(boundary value(X-averaged negative particle concentration [mol.m-3]) / Maximum concentration in negative electrode [mol.m-3], 0.9999999999), 1e-10)) + 1.0 / (-1.0 + maximum(minimum(boundary value(X-averaged negative particle concentration [mol.m-3]) / Maximum concentration in negative electrode [mol.m-3], 0.9999999999), 1e-10))) + (Ambient temperature [K] - Reference temperature [K]) * Negative electrode OCP entropic change [V.K-1] - (-Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m]) * (X-averaged inner SEI thickness [m] + X-averaged outer SEI thickness [m]) * SEI resistivity [Ohm.m]) - SEI open-circuit potential [V]) - (broadcast(Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m])) * (broadcast(X-averaged inner SEI thickness [m]) + broadcast(X-averaged outer SEI thickness [m])) * SEI resistivity [Ohm.m])))'], domains={'primary': ['negative electrode'], 'secondary': ['current collector']})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 601, in process_symbol
    return self._processed_symbols[symbol]
KeyError: Exp(0x3f15ec2a14df9427, function (exp), children=['-SEI growth transfer coefficient * Faraday constant [C.mol-1] / (Ideal gas constant [J.K-1.mol-1] * Ambient temperature [K]) * (broadcast((2.0 * Ideal gas constant [J.K-1.mol-1] * Ambient temperature [K] / Faraday constant [C.mol-1]) * arcsinh(Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m]) / (2.0 * Negative electrode exchange-current density [A.m-2])) + Negative electrode OCP [V] + 1e-06 * (1.0 / (maximum(minimum(boundary value(X-averaged negative particle concentration [mol.m-3]) / Maximum concentration in negative electrode [mol.m-3], 0.9999999999), 1e-10)) + 1.0 / (-1.0 + maximum(minimum(boundary value(X-averaged negative particle concentration [mol.m-3]) / Maximum concentration in negative electrode [mol.m-3], 0.9999999999), 1e-10))) + (Ambient temperature [K] - Reference temperature [K]) * Negative electrode OCP entropic change [V.K-1] - (-Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m]) * (X-averaged inner SEI thickness [m] + X-averaged outer SEI thickness [m]) * SEI resistivity [Ohm.m]) - SEI open-circuit potential [V]) - (broadcast(Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m])) * (broadcast(X-averaged inner SEI thickness [m]) + broadcast(X-averaged outer SEI thickness [m])) * SEI resistivity [Ohm.m]))'], domains={'primary': ['negative electrode'], 'secondary': ['current collector']})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 601, in process_symbol
    return self._processed_symbols[symbol]
KeyError: Multiplication(0x70f092b9efec02a2, *, children=['-SEI growth transfer coefficient * Faraday constant [C.mol-1] / (Ideal gas constant [J.K-1.mol-1] * Ambient temperature [K])', 'broadcast((2.0 * Ideal gas constant [J.K-1.mol-1] * Ambient temperature [K] / Faraday constant [C.mol-1]) * arcsinh(Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m]) / (2.0 * Negative electrode exchange-current density [A.m-2])) + Negative electrode OCP [V] + 1e-06 * (1.0 / (maximum(minimum(boundary value(X-averaged negative particle concentration [mol.m-3]) / Maximum concentration in negative electrode [mol.m-3], 0.9999999999), 1e-10)) + 1.0 / (-1.0 + maximum(minimum(boundary value(X-averaged negative particle concentration [mol.m-3]) / Maximum concentration in negative electrode [mol.m-3], 0.9999999999), 1e-10))) + (Ambient temperature [K] - Reference temperature [K]) * Negative electrode OCP entropic change [V.K-1] - (-Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m]) * (X-averaged inner SEI thickness [m] + X-averaged outer SEI thickness [m]) * SEI resistivity [Ohm.m]) - SEI open-circuit potential [V]) - (broadcast(Current function [A] / (Number of electrodes connected in parallel to make a cell * Electrode width [m] * Electrode height [m]) / Negative electrode thickness [m] / x-average(3.0 * Negative electrode active material volume fraction / Negative particle radius [m])) * (broadcast(X-averaged inner SEI thickness [m]) + broadcast(X-averaged outer SEI thickness [m])) * SEI resistivity [Ohm.m])'], domains={'primary': ['negative electrode'], 'secondary': ['current collector']})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 601, in process_symbol
    return self._processed_symbols[symbol]
KeyError: Multiplication(-0x2dbb906e7115a62c, *, children=['-SEI growth transfer coefficient', 'Faraday constant [C.mol-1] / (Ideal gas constant [J.K-1.mol-1] * Ambient temperature [K])'], domains={})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 601, in process_symbol
    return self._processed_symbols[symbol]
KeyError: Negate(-0x13c4b2571ff2c8d1, -, children=['SEI growth transfer coefficient'], domains={})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 601, in process_symbol
    return self._processed_symbols[symbol]
KeyError: Parameter(0x28ecdcd93390d27b, SEI growth transfer coefficient, children=[], domains={})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/util.py", line 58, in __getitem__
    return super().__getitem__(key)
KeyError: 'SEI growth transfer coefficient'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/asv/benchmark.py", line 1293, in main_run_server
    main_run(run_args)
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/asv/benchmark.py", line 1167, in main_run
    result = benchmark.do_run()
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/asv/benchmark.py", line 573, in do_run
    return self.run(*self._current_params)
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/asv/benchmark.py", line 669, in run
    samples, number = self.benchmark_timing(timer, min_repeat, max_repeat,
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/asv/benchmark.py", line 705, in benchmark_timing
    timing = timer.timeit(number)
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/timeit.py", line 177, in timeit
    timing = self.inner(it, self.timer)
  File "<timeit-src>", line 6, in inner
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/asv/benchmark.py", line 627, in <lambda>
    func = lambda: self.func(*param)
  File "/home/runner/work/PyBaMM/PyBaMM/benchmarks/different_model_options.py", line 152, in time_setup_model
    build_model("Marquis2019", model, "SEI", params)
  File "/home/runner/work/PyBaMM/PyBaMM/benchmarks/different_model_options.py", line 28, in build_model
    param.process_model(model)
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 436, in process_model
    new_rhs[new_variable] = self.process_symbol(equation)
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 603, in process_symbol
    processed_symbol = self._process_symbol(symbol)
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 723, in _process_symbol
    new_child = self.process_symbol(symbol.child)
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 603, in process_symbol
    processed_symbol = self._process_symbol(symbol)
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 714, in _process_symbol
    new_left = self.process_symbol(symbol.left)
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 603, in process_symbol
    processed_symbol = self._process_symbol(symbol)
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 715, in _process_symbol
    new_right = self.process_symbol(symbol.right)
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 603, in process_symbol
    processed_symbol = self._process_symbol(symbol)
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 723, in _process_symbol
    new_child = self.process_symbol(symbol.child)
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 603, in process_symbol
    processed_symbol = self._process_symbol(symbol)
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 715, in _process_symbol
    new_right = self.process_symbol(symbol.right)
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 603, in process_symbol
    processed_symbol = self._process_symbol(symbol)
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 715, in _process_symbol
    new_right = self.process_symbol(symbol.right)
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 603, in process_symbol
    processed_symbol = self._process_symbol(symbol)
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 740, in _process_symbol
    new_children = [self.process_symbol(child) for child in symbol.children]
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 740, in <listcomp>
    new_children = [self.process_symbol(child) for child in symbol.children]
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 603, in process_symbol
    processed_symbol = self._process_symbol(symbol)
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 714, in _process_symbol
    new_left = self.process_symbol(symbol.left)
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 603, in process_symbol
    processed_symbol = self._process_symbol(symbol)
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 714, in _process_symbol
    new_left = self.process_symbol(symbol.left)
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 603, in process_symbol
    processed_symbol = self._process_symbol(symbol)
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 723, in _process_symbol
    new_child = self.process_symbol(symbol.child)
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 603, in process_symbol
    processed_symbol = self._process_symbol(symbol)
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 612, in _process_symbol
    value = self[symbol.name]
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/parameters/parameter_values.py", line 134, in __getitem__
    return self._dict_items[key]
  File "/home/runner/work/PyBaMM/PyBaMM/env/32b4bda701eeaab7fced125b5f96d890/lib/python3.8/site-packages/pybamm/util.py", line 82, in __getitem__
    raise KeyError(f"'{key}' not found. Best matches are {best_matches}")
KeyError: "'SEI growth transfer coefficient' not found. Best matches are ['Edge heat transfer coefficient [W.m-2.K-1]', 'Positive electrode charge transfer coefficient', 'Negative electrode charge transfer coefficient']"
```

Reply:
A way around it would be to add `{'SEI growth transfer coefficient': 0.5}` to the parameters